package.json

```diff
@@ -3,6 +3,10 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@emotion/react": "^11.9.0",
```

Comment:
Do we actually use this?

Reply:
I dont beleive we do, it may have snuck in with "MUI".

Reply:
Removed  ` @emotion/react": "^11.9.0", `